### DISCUSSION TITLE
LocalSend

### DISCUSSION THREAD

**Discussion by Pamatos** [localsend/localsend#2141] [L-Français]
Bonjour,
Je viens d'installer LocalSend sur mon MacIntel (macOS 15) en le téléchargeant depuis App Store.
Pas trouvé comment faire reconnaître mon téléphone (Samsung/Android) connecté en USB-C !
Seul le Wifi m'était proposé et là, j’étais un peu perdu.
Dois-je aussi télécharger LocalSend depuis  [Google Play](https://play.google.com/store/apps/details?id=org.localsend.localsend_app&hl=fr) et l'installer également sur le téléphone mobile…
Je n'ai pas trouvé de manuel…
Merci à l'avance de l'aide.

**Comment by fiorens** [comment]
Bonjour,
Si j'ai bien compris le logiciel implémente lui même tout ce qu'il faut pour envoyer/recevoir un fichier. Il faut donc l'installer sur votre MAC mais aussi sur votre Samsung. Le lancer des deux côtés également.
Bien à vous.
